editor/scene_create_dialog.cpp

```diff
@@ -131,6 +131,22 @@ void SceneCreateDialog::update_dialog() {
 	root_name = root_name_edit->get_text().strip_edges();
 	if (root_name.is_empty()) {
 		root_name = scene_name.get_file().get_basename();
+
+		// Set the placeholder text.
```

Comment:
This block not only sets the placeholder, but it also generates the root name and sets the warning, so the comment is a bit confusing in my opinion.